CHANGES.rst

```diff
@@ -6,6 +6,9 @@ Bug Fixes
 
 - Fixed ``ValidationError`` during ``AmiOIModel.update`` [#234]
 
+- Adding emicorr datamodel and schema, as well as
+ corresponding completion and reference file keywords [#200]
```

Comment:
minor formatting issue: 2nd line needs to match the indent of the line above (i.e., the "c" needs to be directly under the "A")

Reply:
This still needs fixing.